src/game.cpp

```diff
@@ -8098,45 +8194,45 @@ void game::examine( const tripoint &examp )
         Creature *c = critter_at(examp);
         monster *mon = dynamic_cast<monster *>(c);
 
-        if( mon != nullptr && mon->has_effect("pet") ) {
+        if ( mon != nullptr && mon->has_effect("pet") ) {
             if (pet_menu(mon)) {
                 return;
             }
         }
 
-        npc *np = dynamic_cast<npc*>( c );
-        if( np != nullptr ) {
-            if( npc_menu( *np ) ) {
+        npc *np = dynamic_cast<npc*>(c);
+        if (np != nullptr) {
+            if (npc_menu(*np)) {
                 return;
             }
         }
     }
 
-    if( !m.tr_at( examp ).is_null() ) {
-        iexamine::trap(&u, &m, examp);
+    if ( !m.tr_at(examp).is_null() ) {
+        iexamine::trap( &u, &m, examp );
         draw_ter();
     }
 
     // In case of teleport trap or somesuch
-    if( player_pos != u.pos() ) {
+    if ( player_pos != u.pos( )) {
```

Comment:
Lots and lots of needless whitespace changes. `doc/CODE_STYLE.txt` states there should be spaces inside the `()`, example from there:
```C++
if( arg1 < 5 ) {
```
The current code seems to confirm to that.